src/Uno.UWP/ApplicationModel/Resources/ResourceLoader.cs

```diff
@@ -48,6 +48,23 @@ public ResourceLoader()
 
 		public string GetString(string resource)
 		{
+			
+			// "/[file]/[name]" format support
+			if(resource.ElementAt(0) == '/')
+			{
+				int iInd;
+				string sFile, sId;
+				sId = resource.Substring(1);
+				iInd = sId.IndexOf("/");
```

Comment:
Use the startIndex parameter of `string.IndexOf` to avoid creating a sub-string
```suggestion
				iInd = sId.IndexOf("/", 1);
```

Reply:
With substring:
sId = resource.Substring(1);
iInd = sId.IndexOf("/");
...
sFile = sId.Substring(0, iInd);

Without substring:
sId = resource;
iInd = sId.IndexOf("/",1);
...
sFile = sId.Substring(1, iInd);

seems same...

Maybe:
iInd = resource.IndexOf("/",1);
...
sFile = resource.Substring(1, iInd);

be better (if it cannot be with substring)

Reply:
The use of `IndexOf` with the start index avoid the creation of an intermediate string starting at index 1.